src/cdomains/pentagon/intv.ml

```diff
@@ -0,0 +1,161 @@
+open ZExt
+open Batteries
+open StringUtils
+
+(**
+   Stores functions and types for single intervals $\mathbb{Z}^\infty$
+   according to the pentagon domains semantics. Beware, this module is NOT generic.
+*)
+module Intv =
+struct
+  include ZExtOps
```

Comment:
This is almost a lattice except you have named (some) operations differently? Any reasons for that?